crates/zksync/compiler/src/zksolc/compile.rs

```diff
@@ -299,7 +299,7 @@ impl ZkSolc {
                 ))?;
 
                 // Step 4: Build Compiler Arguments
-                let comp_args = self.build_compiler_args(&contract_path, &self.project.solc);
+                let comp_args = self.build_compiler_args(&contract_path, &solc);
```

Comment:
iirc this was done due to a different compile error in the beginning of the project but if everything works then it was probably superflous